src/gui/app/services/activity-feed.service.js

```diff
@@ -24,7 +24,7 @@
                 }
 
                 const allowedEvents = settingsService.getAllowedActivityEvents();
-                if (!allowedEvents.includes(`${activity.source.id}:${activity.event.id}`)) {
+                if (!allowedEvents.includes(`${activity.source.id}:${activity.event.id}`) && `${activity.source.id}:${activity.event.id}` !== "firebot:activity-feed-alert") {
                     return;
                 }
```

Comment:
Given the changes we made to pass along event settings, we can remove the hardcoded check and do this instead:
```js
if (!activity.event.forceAllow && !allowedEvents.includes(`${activity.source.id}:${activity.event.id}`)) {
    return;
}
```